build.sbt

```diff
@@ -82,13 +96,6 @@ lazy val circe = project
   .settings(commonSettings)
   .settings(
     name := "gsp-graphql-circe",
-    libraryDependencies ++= Seq(
-      "org.tpolecat"      %% "atto-core"              % attoVersion,
-      "org.typelevel"     %% "cats-core"              % catsVersion,
-      "io.circe"          %% "circe-core"             % circeVersion,
-      "io.circe"          %% "circe-optics"           % circeOpticsVersion,
-      "io.circe"          %% "circe-parser"           % circeVersion
-    )
```

Comment:
These dependencies all arrive transitively so I removed them as a simplification. Same in a few places below.